csu

```diff
@@ -357,8 +354,14 @@ ci_docs() {
   dev_docs
 }
 
+ci_load_content() {
+  cmd_update
+}
+
 cmd_ci() {
   cmd_start
+  dev_static
```

Comment:
Would it be better to do this within the tests that require it? For example, style checker doesn't require this.